python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -242,6 +216,25 @@ def remove(self):
 
         return
 
+    def put(self):
+	"""
+        For the LocalFile it makes no sense to do anything here
```

Comment:
This comment does not match the fact that you *are* doing something in this function. Also, this sort of thing is better put in a comment that in a docstring.